---
layout: default
title: Home
---


<div class="box">
            <h2>Welcome to HiTSeq 2024<!-- Place this tag where you want the +1 button to render. -->
            </h2>
            HiTSeq is the yearly gathering of the ISCB Community of Special
            Interest in High Throughput Sequencing algorithms. This meeting
            takes the form of a topical track at the <a href="https://www.iscb.org/ismb2024"
              target="_top">ISMB 2024 Conference</a>, and is devoted to the
            latest advances in computational techniques for the analysis of
            high-throughput sequencing (HTS) data. It provides a forum for in
            depth presentations of novel algorithms, analysis methods, and
            methods enabling applications in multiple areas of biology that HTS
            is transforming.
</div>

<div id="col1">
            <h3>Call for Submissions</h3>
            <ul class="section-list">
              <li class="first"> <br />
              </li>
              <li>
                <h4>Types of Contributions</h4>
                HiTSeq seeks full-length original paper submissions to be
                peer-reviewed. If accepted, these contributions will be
                presented at the conference and published as part of the ISMB
                proceedings. Short abstract submissions will be also accepted to
                be selected for either brief oral presentations or posters. </li>
              <li>
                <h4>Focus of Submissions</h4>
                <span> HiTSeq focus is at the intersection of the sequencing
                  data and the applications this technology enables. Due to the
                  massively parallel data generation, the huge volumes that the
                  latest platforms deliver, and the continually decaying cost,
                  this technology has been using in novel ways, well beyond its
                  nucleic acid sequencing capabilities. New single molecule
                  sequencing platforms are starting to deliver very long reads
                  that enable discovery of new forms of genome variation. With
                  these innovations new methodological aspects and errors are
                  generated and need to be managed.
                  <p></p>
                </span><span style="font-size:10.5pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;;color:#595959"><b></b></span></li>
            </ul>
</div>

<div id="col2"><br />
            <br />
            <ul class="section-list">
              <li class="first"> <br />
              </li>
              <li>
                <h4>Topics of Interest<span style="font-size:10.5pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;;color:#595959"><span
                      style="color: #212121;"></span></span></h4>
                <span style="font-size:10.5pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;;color:#595959">
                  Applications of HTS range from genome sequencing and assembly,
                  genetic variation analysis, cancer genomics, transcriptomics,
                  epigenetics, metagenomics, and general sequence analysis
                  algorithms, in particular those where new sequence
                  technologies such as single molecule/long reads are used and
                  novel data analysis modalities are applied. We thus seek
                  contributions where algorithms and methods that take HTS data
                  and convert this into a biological signal.Â  <br />
                  <p> </p>
                </span> </li>
              <li class="last"><br />
              </li>
            </ul>
            <p><br />
            </p>
            <ul class="section-list">
            </ul>
</div>
          
