---
layout: default
title: Schedule 2023
---


<div class="box">
  <h2>Schedule of HiTSeq 2023<br />July 25-26, 2023, Lyon, France</h2>


  <h3>Keynote speakers</h3>
  <br />
  <table width="100%" border="1">
    <tbody>
      <tr>
        <td><img src="images/korbel.jpg" alt="Jan Korbel" title="Jan Korbel"
                 style="width: 220px; height: 220px;" /><br />
        </td>
        <td><a href="https://www.embl.org/groups/korbel/">Jan Korbel</a><br />
          Head of Data Science<br />
          European Molecular Biology Laboratory, Germany</td>
      </tr>
      <tr>
        <td><img src="images/irene.jpg" alt="Irene Papatheodorou" title="Irene Papatheodorou" /><br />
        </td>
        <td><a href="https://www.ebi.ac.uk/people/person/irene-papatheodorou/">Irene
            Papatheodorou</a><br />
          <span style="color: #222222;"></span>Team Leader<br />
          European Bioinformatics Institute, UK</td>
      </tr>
    </tbody>
  </table>
  <p><br />
  </p>
  <h3>July 25, 2023</h3>
  <table style="width: 100%;" border="0">
    <tbody>
      <tr>
        <td><strong>10:30</strong></td>
        <td><br />
        </td>
        <td><strong>Welcome remarks</strong></td>
      </tr>
      <tr>
        <td><strong>10:35<br />
            <br />
            <br />
        </strong></td>
        <td><br />
        </td>
        <td><strong>Keynote: </strong><strong>Using single cell data to
            understand disease and cell type differences across species<br />
            Irene Papatheodorou,  European Bioinformatics Institute<br />
        </strong></td>
      </tr>
      <tr>
        <td><strong>11:30<br />
            <br />
            <br />
        </strong></td>
        <td><br />
        </td>
        <td><strong>Detecting Chromosomal Translocations using Augmented
            Genome Sequence Graphs.</strong> Alister D'Costa, Philip <span
								       style="color: black;">Zuzarte</span>, Michael Molnar, Tracy
          Murphy, Mark Minden, Yun William Yu, <br />
          Jared Simpson</td>
      </tr>
      <tr>
        <td><strong>11:50<br />
            <br />
        </strong></td>
        <td><br />
        </td>
        <td><strong>VeChat: Correcting errors in long reads using
            variation graphs.</strong> Xiao Luo, Xiongbin Kang,
          Alexander Schoenhuth</td>
      </tr>
      <tr>
        <td><strong>12:10<br />
            <br />
        </strong></td>
        <td><br />
        </td>
        <td><strong>RawHash: Enabling Fast and Accurate Real-Time
            Analysis of Raw Nanopore Signals for Large Genomes.</strong>
          Can Firtina, Nika Mansouri Ghiasi, Joel Lindegger, Gagandeep
          Singh, Meryem Banu Cavlak, Haiyu Mao, Onur Mutlu</td>
      </tr>
      <tr>
        <td><strong><br />
        </strong></td>
        <td><br />
        </td>
        <td><br />
        </td>
      </tr>
      <tr>
        <td><strong>12:30</strong></td>
        <td><br />
        </td>
        <td><strong>Lunch break</strong></td>
      </tr>
      <tr>
        <td><strong><br />
        </strong></td>
        <td><br />
        </td>
        <td><br />
        </td>
      </tr>
      <tr>
        <td><strong>13:40<br />
            <br />
        </strong></td>
        <td><br />
        </td>
        <td><strong>ALLSTAR: Inference of ReliAble CausaL RuLes between
            Somatic MuTAtions and CanceR Phenotypes</strong>. Dario
          Simionato, Antonio Collesei, Federica Miglietta, Fabio Vandin</td>
      </tr>
      <tr>
        <td><strong>14:10<br />
            <br />
        </strong></td>
        <td><br />
        </td>
        <td><strong>Estimate mutational signature exposure from sparse
            clinical sequencing data.  </strong><span style="color: #656565;">Arnab
            Chakrabarti, Hiroshi Hamano, Lancelot Seillier, Kjong-Van
            Lehmann</span><strong></strong></td>
      </tr>
      <tr>
        <td><strong>14:30<br />
            <br />
            <br />
        </strong></td>
        <td><br />
        </td>
        <td><strong>PEKORA: High-Performance 3D Genome Reconstruction
            Using K-th Order Spearman's Rank Correlation Approximation. </strong>
          Yeremia Gunawan Adhisantoso, Jan Voges, Jörn Ostermann</td>
      </tr>
      <tr>
        <td><strong>14:50<br />
            <br />
            <br />
        </strong></td>
        <td><br />
        </td>
        <td><strong style="color: black;">Genetic interactions between
            translesion DNA synthesis enzymes in cancer.</strong> Aleix
          Bayona-Feliu, Marcos Moreno, Sergio Marín-Edo, Marcel
          McCullough, Miguel-Martín Álvarez, Fran Supek</td>
      </tr>
      <tr>
        <td><strong>15:10<br />
            <br />
            <br />
        </strong></td>
        <td><br />
        </td>
        <td><strong style="color: black;">Deep statistical modelling of
            nanopore sequencing translocation times reveals latent non-B
            DNA structures.</strong> Marjan Hosseini, Aaron Palmer,
          William Manka, Patrick GS Grady, Venkata Patchigolla, Jinbo
          Bi, Rachel O'Neill, Zhiyi Chi, Derek Aguiar</td>
      </tr>
      <tr>
        <td><strong><br />
        </strong></td>
        <td><br />
        </td>
        <td><br />
        </td>
      </tr>
      <tr>
        <td><strong>15:30</strong></td>
        <td><br />
        </td>
        <td><strong>Coffee break</strong></td>
      </tr>
      <tr>
        <td><strong><br />
        </strong></td>
        <td><br />
        </td>
        <td><br />
        </td>
      </tr>
      <tr>
        <td><strong>16:00<br />
            <br />
        </strong></td>
        <td><br />
        </td>
        <td><strong>SuperSampler: efficient scaled sketches for
            metagenomics and extensive genomics compositional analysis.
          </strong>Timothé Rouzé, Camille Marchet, Antoine Limasset</td>
      </tr>
      <tr>
        <td><strong>16:20<br />
            <br />
        </strong></td>
        <td><br />
        </td>
        <td><strong style="color: black;">Locality-Preserving Minimal
            Perfect Hashing of K-Mers.</strong> Giulio Ermanno Pibiri,
          Yoshihiro Shibuya, Antoine Limasset</td>
      </tr>
      <tr>
        <td><strong>16:40<br />
            <br />
        </strong></td>
        <td><br />
        </td>
        <td><strong style="color: black;">Building a Pangenome Alignment
            Index via Recursive Prefix-Free Parsing.</strong> Marco
          Oliva, Travis Gagie, Christina Boucher</td>
      </tr>
      <tr>
        <td><strong>17:00<br />
        </strong></td>
        <td><br />
        </td>
        <td><strong style="color: black;">Effects of Spaced k-mers on
            Alignment-Free Genotyping</strong><span style="color: black;">.</span>
          Hartmut Häntze, Paul Horton</td>
      </tr>
      <tr>
        <td><strong><span style="color: #656565;">1</span>7:20<br />
        </strong></td>
        <td> <br />
        </td>
        <td><strong>Seeding with Minimized Subsequence. </strong>Xiang
          Li, Qian Shi, Ke Chen, Mingfu Shao</td>
      </tr>
      <tr>
        <td><strong>17:40<br />
            <br />
        </strong></td>
        <td><br />
        </td>
        <td><strong>Scalable sequence database search using Partitioned
            Aggregated Bloom Comb-Trees.</strong> Camille Marchet,
          Antoine Limasset</td>
      </tr>
    </tbody>
  </table>
  <p><br />
  </p>
  <h1> </h1>
  <br />
  <h3>July 26, 2023</h3>
  <br />
  <table style="width: 100%;" border="0">
    <tbody>
      <tr>
        <td><strong>10:35<br />
            <br />
        </strong> </td>
        <td><br />
        </td>
        <td><strong>Keynote: </strong><strong>Deciphering genomic
            disease mechanisms via single-cell &amp; single-molecule
            sequencing<strong><br />
          </strong></strong><strong>Jan Korbel,  European Molecular
            Biology Laboratory<br />
        </strong> </td>
      </tr>
      <tr>
        <td><strong>11:30<br />
            <br />
        </strong> </td>
        <td> <br />
        </td>
        <td><strong>Coriolis: Enabling metagenomic classification on
            lightweight mobile devices.</strong> Andrew Mikalsen,
          Jaroslaw Zola<br />
        </td>
      </tr>
      <tr>
        <td><strong>11:50<br />
            <br />
        </strong> </td>
        <td> <br />
        </td>
        <td><strong>Metabuli: sensitive and specific metagenomic
            classification through a novel joint analysis of amino-acid
            and DNA sequences.</strong> Jaebeom Kim, Martin Steinegger<br />
        </td>
      </tr>
      <tr>
        <td><strong>12:10<br />
            <br />
        </strong> </td>
        <td><br />
        </td>
        <td><strong>HaploDMF: viral haplotype reconstruction from long
            reads via deep matrix factorization.</strong>  Dehan Cai,
          Jiayu Shang</td>
      </tr>
      <tr>
        <td><strong> </strong><br />
        </td>
        <td><br />
        </td>
        <td><br />
        </td>
      </tr>
      <tr>
        <td><strong>12:30<br />
        </strong> </td>
        <td><br />
        </td>
        <td><strong>Lunch break</strong><br />
        </td>
      </tr>
      <tr>
        <td><strong><br />
        </strong></td>
        <td><br />
        </td>
        <td><br />
        </td>
      </tr>
      <tr>
        <td><strong>13:50<br />
        </strong></td>
        <td><br />
        </td>
        <td><strong>SVJedi-graph: improving the genotyping of close and
            overlapping Structural Variants with long reads using a
            variation graph.</strong> Sandra Romain, Claire Lemaitre</td>
      </tr>
      <tr>
        <td><strong>14:10<br />
        </strong></td>
        <td><br />
        </td>
        <td><strong>Taxor: Fast and space-efficient taxonomic
            classification of long reads. </strong>Jens-Uwe Ulrich, <br />
          Bernhard Renard</td>
      </tr>
      <tr>
        <td><strong>14:30<br />
            <br />
        </strong> </td>
        <td><br />
        </td>
        <td><strong style="color: black;">Leveraging Evolutionary
            Constraints to Refine Somatic Variant Calls from Single-Cell
            Sequencing Data.</strong> Gryte Satas, Matthew A. Myers,
          Seongmin Choi, Sohrab Shah<br />
        </td>
      </tr>
      <tr>
        <td><strong>14:50<br />
            <br />
            <br />
        </strong> </td>
        <td><br />
        </td>
        <td><strong>Themisto: a scalable colored k-mer index for
            sensitive pseudoalignment against hundreds of thousands of
            bacterial genomes.</strong> Jarno Alanko, Simon Puglisi,
          Tommi Mäklin, <br />
          Jaakko Vuohtoniemi, </td>
      </tr>
      <tr>
        <td><strong>15:10<br />
            <br />
            <br />
            <br />
        </strong> </td>
        <td><br />
        </td>
        <td><strong>A multi-locus approach for accurate variant calling
            in low-copy repeats using whole-genome sequencing.</strong>
          Timofey Prodanov, Heinrich Heine University, Duesseldorf
          40225, Germany, Germany<br />
          Vikas Bansal<br />
        </td>
      </tr>
      <tr>
        <td><strong><br />
        </strong> </td>
        <td><br />
        </td>
        <td><br />
        </td>
      </tr>
      <tr>
        <td><strong>15:30<br />
        </strong> </td>
        <td><br />
        </td>
        <td><strong>Coffee break</strong><br />
        </td>
      </tr>
      <tr>
        <td><strong><br />
        </strong> </td>
        <td><br />
        </td>
        <td><br />
        </td>
      </tr>
      <tr>
        <td><strong>16:00<br />
            <br />
            <br />
        </strong> </td>
        <td><br />
        </td>
        <td><strong>Variational Inference for Single-Cell Transcriptome
            with DNA Barcoding Reconstructs Unobserved Cell States and
            Differentiation Trajectories. </strong>Koichiro Majima,
          Kodai Minoura, <br />
          Yasuhiro Kojima, Teppei Shimamura <br />
        </td>
      </tr>
      <tr>
        <td><strong>16:20<br />
            <br />
        </strong> </td>
        <td><br />
        </td>
        <td><strong>GAN-based Data Augmentation for Transcriptomics:
            Survey and Comparative Assessment. </strong>Alice Lacan,
          Michele Sebag, Blaise Hanczar<br />
        </td>
      </tr>
      <tr>
        <td><strong>16:40<br />
            <br />
        </strong> </td>
        <td><br />
        </td>
        <td><strong>Visualizing Spatial Transcriptomics with U-CIE Color
            Encoding.</strong> Mikaela Koutrouli, Radha Swaminathan,
          Jeremy Edwards, Lars Juhl Jensen<br />
        </td>
      </tr>
      <tr>
        <td><strong>17:00<br />
            <br />
            <br />
            <br />
        </strong> </td>
        <td><br />
        </td>
        <td><strong>CellFromSpace: A versatile tool for spatial
            transcriptomic data analysis through reference-free
            deconvolution and guided cell type/activity annotation.</strong>
          Corentin Thuilliez, <br />
          Maria Eugenia Marques Da Costa, Nathalie Gaspar, Pierre
          Khneisser, Gael Moquin-Beaudry, Jean-Yves Scoazec, Antonin
          Marchais<br />
        </td>
      </tr>
      <tr>
        <td><strong>17:20<br />
            <br />
        </strong></td>
        <td><br />
        </td>
        <td><strong>demuxSNP: supervised demultiplexing of scRNAseq data
            using cell hashing and SNPs.</strong> Michael P Lynch, Yufei
          Wang, Laurent Gatto, Aedin C Culhane</td>
      </tr>
      <tr>
        <td><strong>17:40<br />
            <br />
            <br />
        </strong></td>
        <td><br />
        </td>
        <td><strong>Foreign RNA spike-ins enable accurate
            allele-specific expression analysis at scale. </strong>Asia
          Mendelevich, Saumya Gupta, Aleksei Pakharev, Athanasios
          Teodosiadis, Andrey Mironov, Alexander Gimelbrant</td>
      </tr>
    </tbody>
  </table>
  <p><br />
  </p>
  <br />
</div>

